DOCS-2565: Document Magento 2 order lifetime for Second Chance (#1681)
MultiSafepay/docs: content/payments/integrations/ecommerce-platforms/magento2/faq/setting-order-lifetimes-second-chance.md

read_more: "."
---

[Second Chance](/payments/boost/second-chance/) emails are sent 1 hour and 24 hours after the order was created. By default, the status of Magento 2 orders changes from **Pending payment** to **Cancelled** after 8 hours (480 minutes).

If the customer pays via the **second** email (24 hours later), the payment is processed but the transaction update may not be handled correctly in Magento 2 because the order has expired. This may cause issues with external services (e.g. ERP/inventory management), if items are low in stock, or for one-off products like antiques.

To avoid this, you must match the order lifetime to the MultiSafepay payment link lifetime in your [backend](/getting-started/glossary/#backend).

We recommend setting the lifetimes to 2 days (2880 minutes) to allow enough time for the customer to pay, but to avoid issues with external services. 

## Setting the Magento 2 order lifetime

1. Sign in to your backend.
2. Go to **Stores** > **Configuration** > **Sales** > **Sales** > **Orders cron settings**.
3. Select **Pending payment order lifetime (minutes)** and set to **2880 minutes**.

## Setting the MultiSafepay payment link lifetime

> 1. From your Magneto 2 admin panel, go to **Stores** > **Configuration** > **MultiSafepay** > **Payment gateways**.

Reviewer: common typo Magneto should be Magento.

> 1. From your Magneto 2 admin panel, go to **Stores** > **Configuration** > **MultiSafepay** > **Payment gateways**.

Author: ```suggestion
1. From your Magento 2 admin panel, go to **Stores** > **Configuration** > **MultiSafepay** > **Payment gateways**.
```

---
title : "Setting order lifetimes for Second Chance"
meta_title: "Magento 2 plugin FAQ - Setting order lifetimes for Second Chance - MultiSafepay Docs"
meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API reference, SDKs, and wrappers. Get support."
layout: "faqdetail"
read_more: "."
---

> [Second Chance](/payments/boost/second-chance/) emails are sent 1 hour and 24 hours after the order was created. By default, Magento 2 orders are valid for 2 hours.

Reviewer: I would say instead that Magento 2 orders that have the 'pending payment' state are being set to the 'cancelled' state, instead of saying that they are not 'valid' anymore, because there is no such thing as a 'valid' order in Magento.

Also, the standard pending payment lifetime of an order is actually 8 hours (480 minutes) and not 2. See the Magento devdocs for more information:
https://docs.magento.com/user-guide/sales/order-pending-payment-lifetime.html

Author: @Vinod-MultiSafepay 
[Second Chance](/payments/boost/second-chance/) emails are sent 1 hour and 24 hours after the order was created. By default, the status of Magento 2 orders changes from **Pending payment** to **Cancelled** after 8 hours (480 minutes).

> [Second Chance](/payments/boost/second-chance/) emails are sent 1 hour and 24 hours after the order was created. By default, Magento 2 orders are valid for 2 hours.

Author: ```suggestion
[Second Chance](/payments/boost/second-chance/) emails are sent 1 hour and 24 hours after the order was created. By default, the status of Magento 2 orders changes from **Pending payment** to **Cancelled** after 8 hours (480 minutes).
```

> If the customer pays via the **second** email (24 hours later), the payment is processed but the transaction update isn't handled correctly in Magento 2 because the order has expired.

Reviewer: This only happens in _some_ cases, when there isn't any stock left for the product for example or as stated further down the article, when a merchant is using an external service.

Author: @Vinod-MultiSafepay 
If the customer pays via the **second** email (24 hours later), the payment is processed but the transaction update may not be handled correctly in Magento 2 because the order has expired. This may cause issues with external services (e.g. ERP/inventory management), if items are low in stock, or for one-off products like antiques.

Reviewer: Yes!

> If the customer pays via the **second** email (24 hours later), the payment is processed but the transaction update isn't handled correctly in Magento 2 because the order has expired.

Author: ```suggestion
If the customer pays via the **second** email (24 hours later), the payment is processed but the transaction update may not be handled correctly in Magento 2 because the order has expired. This may cause issues with external services (e.g. ERP/inventory management), if items are low in stock, or for one-off products like antiques.
```

To avoid this, you must match the order lifetime to the MultiSafepay payment link lifetime in your [backend](/getting-started/glossary/#backend).

> We recommend setting the lifetimes to 2 days (2880 minutes):

Author: ```suggestion
We recommend setting the lifetimes to 2 days (2880 minutes) to allow enough time for the customer to pay, but to avoid issues with external services. 
```

> - Shorter gives the the customer too little time to complete the payment. 

Reviewer: typo here with 2 times 'the' after each other :)

> - Shorter gives the the customer too little time to complete the payment. 

Author: ```suggestion
- Shorter gives the customer too little time to complete the payment. 
```

> - Shorter gives the the customer too little time to complete the payment. 

Author: ```suggestion
```

> - Longer can cause issues with external services (e.g. ERP/inventory management), which may be problematic when items are low in stock or for one-off products like antiques.

Author: ```suggestion
```
